Makefile.in

```diff
@@ -347,6 +347,7 @@ libaddon: exports
 	$(MAKE) -C lib/addons/library.kodi.peripheral
 	$(MAKE) -C lib/addons/library.xbmc.pvr
 	$(MAKE) -C lib/addons/library.kodi.inputstream
+	PREFIX=$(DESTDIR)$(prefix) $(MAKE) -C xbmc/addons/kodi-addon-dev-kit
```

Comment:
you should optionally include the top Makefile.include in(-include ...) in the kodi-addon-dev-kit/Makefile
this way you don't need to pass in those variables